setup.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 # coding=utf-8
```

Comment:
As a side note, since UTF-8 is the default in Python 3, we don't need these encoding cookies any more.

Reply:
Missed removing this one, along with a couple others in `sopel.py` and `pytest_run.py` noted separately, and there's a different style of encoding cookie in `docs/source/conf.py` that could be deleted.

Several test files also define a `MOCK_MODULE_CONTENT` or similar that contains the coding cookie.